build/Build.Steps.Windows.cs

```diff
@@ -177,4 +179,20 @@ partial class Build
 
             AssemblyRedirectionSourceGenerator.Generate(netFxAssembliesFolder, generatedSourceFile);
         });
+
+    Target InstallNetFxAssembliesGAC => _ => _
+        .Unlisted()
+        .Before(RunManagedTests)
```

Comment:
Could you please convert this condition to `.After(InstallNetFxAssembliesGAC )` on `RunManagedTests`?

Reply:
Found better place for the `After` method.